.travis.yml

```diff
@@ -29,26 +29,46 @@ matrix:
     - env: MODE=model PR_ONLY=true
     - env: MODE=BreakingChange PR_ONLY=true
     - env: MODE=azurebot PR_ONLY=true
+    - env: MODE=liveValidation PR_ONLY=true
 before_install:
   - docker pull lmazuel/swagger-to-sdk
-  - python -c "import os; print('\n'.join(v for v in os.environ.keys() if v.startswith('TRAVIS')))" > /tmp/env_file
-  # Required for installing dotnet 2.0.0 according to https://www.microsoft.com/net/core#linuxubuntu
-  - curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+  - >-
+    python -c "import os; print('\n'.join(v for v in os.environ.keys() if
+    v.startswith('TRAVIS')))" > /tmp/env_file
+  - >-
+    curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor >
+    microsoft.gpg
   - sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
-  - sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-trusty-prod trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
+  - >-
+    sudo sh -c 'echo "deb [arch=amd64]
+    https://packages.microsoft.com/repos/microsoft-ubuntu-trusty-prod trusty
+    main" > /etc/apt/sources.list.d/dotnetdev.list'
   - sudo apt-get update
   - sudo apt-get install libunwind8 libicu52 -y
   - sudo apt-get install dotnet-sdk-2.0.0 -y
 install:
   - npm install
 script:
-  - DOCKER_CMD="docker run --rm --env-file /tmp/env_file -e GH_TOKEN -v $PWD:/git-restapi/ lmazuel/swagger-to-sdk"
-  - if [[ $MODE == 'python' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-python --pr-repo-id Azure/azure-sdk-for-python -o master -v; fi
-  - if [[ $MODE == 'node' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-node --pr-repo-id Azure/azure-sdk-for-node -o master -v; fi
-  - if [[ $MODE == 'ruby' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-ruby --pr-repo-id Azure/azure-sdk-for-ruby -o master -v; fi
-  - if [[ $MODE == 'syntax' ]]; then npm test -- test/syntax.js; fi
-  - if [[ $MODE == 'linter' ]]; then npm test -- test/linter.js; fi
-  - if [[ $MODE == 'semantic' ]]; then npm test -- test/semantic.js; fi
-  - if [[ $MODE == 'model' ]]; then npm test -- test/model.js; fi
-  - if [[ $MODE == 'BreakingChange' ]]; then node -- scripts/breaking-change.js; fi
-  - if [[ $MODE == 'azurebot' ]]; then node scripts/momentOfTruth.js; fi
+  - >-
+    DOCKER_CMD="docker run --rm --env-file /tmp/env_file -e GH_TOKEN -v
+    $PWD:/git-restapi/ lmazuel/swagger-to-sdk"
+  - >-
+    if [[ $MODE == 'python' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-python
+    --pr-repo-id Azure/azure-sdk-for-python -o master -v; fi
+  - >-
+    if [[ $MODE == 'node' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-node
+    --pr-repo-id Azure/azure-sdk-for-node -o master -v; fi
+  - >-
+    if [[ $MODE == 'ruby' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-ruby
+    --pr-repo-id Azure/azure-sdk-for-ruby -o master -v; fi
+  - 'if [[ $MODE == ''syntax'' ]]; then npm test -- test/syntax.js; fi'
```

Comment:
why do we need quotes around these lines? and double quotes for the MODE values?